lib/auth/auth_with_roles.go

```diff
@@ -1577,7 +1601,9 @@ func (a *ServerWithRoles) ListUnifiedResources(ctx context.Context, req *proto.L
 		}()
 	}
 
-	resourceChecker, err := a.newResourceAccessChecker(types.KindUnifiedResource)
+	actionVerbs := []string{types.VerbList, types.VerbRead}
```

Comment:
i would model these `verbList` with previous implementations because we don't want to deny access to something we gave access for previously (example is `nodes`, for some reason, it only checks for rbac `list`, but now we require `read` here as well, so access may break for some)

Reply:
when did we release this feature again? i'm a little worried about the implication of this bug for previous v14 versions. not sure what the right protocol is, or perhaps i'm just over-thinking